src/main/resources/assets/api-docs.yaml

```diff
@@ -3824,39 +3820,6 @@ components:
         displayName:
           description: System assigned create user display name
           type: string
-    DarCollection:
```

Comment:
Is it possible to link the yaml file so that the model schema is still listed under the Schema subsection?

Reply:
Good question ... testing that out.

Reply:
Yes, that does work, thanks for the suggestion!